py/manylinux.sh

```diff
@@ -7,21 +7,23 @@ yum -y -q install gcc libffi-devel
 
 export PKG_CONFIG_PATH=/usr/lib/x86_64-linux-gnu/pkgconfig/
 
-# Install Rust
-curl https://sh.rustup.rs -sSf | sh -s -- -y
-export PATH=~/.cargo/bin:$PATH
+# Install Rust if we have to build the lib inside of this container
+if [ -z $SKIP_RELAY_LIB_BUILD ]; then
```

Comment:
If I understand correctly, `-z` returns true only if `$SKIP_RELAY_LIB_BUILD` is null (length zero). Should we consider truthy/falsy cases, such as `0` or `1` and `true` or `false`?

Reply:
I don't think we  have to care  here about this. It should be fine to know if it's set or not.  
But also this script can be simplified and the part about the cargo removed - maybe in this PR or as a followup if the proposed changes are accepted

Reply:
Nah, it's fine.